docs/how-to/integrate_the_sdcore_user_plane_with_cos.md

```diff
@@ -0,0 +1,50 @@
+# Integrate the SD-CORE User Plane with COS
```

Comment:
If people need to find this through an index, it's not obvious what COS is. I'd explicitly use the word Observability in the index and main title.